docker-compose.yml

```diff
@@ -10,6 +10,15 @@ services:
     env_file:
       - .env
     ports:
-      - "27017:27017"
+      - '27017:27017'
     volumes:
       - ./mongo-data:/data/db
+  app:
+    container_name: aremis-web
+    restart: always
+    build: .
+    env_file:
+      - .env
+    depends_on:
+      - mongodb
+    command: ./wait.sh ${DEFAULT_EMAIL} ${DEFAULT_PASS}
```

Comment:
no need to have custom command. just add to the Dockerfile to start the entrypoint.sh